docs/data/data-grid/pivoting/pivoting.md

```diff
@@ -2,16 +2,138 @@
 title: Data Grid - Pivoting
 ---
 
-# Data Grid - Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')🚧
+# Data Grid - Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
 
-<p class="description">Rearrange columns and values to view data from multiple perspectives.</p>
+<p class="description">Rearrange rows and columns to view data from multiple perspectives.</p>
 
-:::warning
-This feature isn't available yet, but it is planned—you can 👍 upvote [this GitHub issue](https://github.com/mui/mui-x/issues/214) to help us prioritize it.
-Please don't hesitate to leave a comment there to describe your needs, especially if you have a use case we should address or you're facing specific pain points with your current solution.
+The Data Grid Premium's pivoting feature lets users transform the data in their grid by reorganizing rows and columns, creating dynamic cross-tabulations of data.
+This makes it possible to analyze data from different angles and gain insights that would be difficult to see in the default grid view.
+
+:::success
+If you're new to pivoting, check out the [Understanding pivoting](/x/react-data-grid/pivoting-explained/) page to learn what pivoting is and how it works through interactive examples.
 :::
 
```

Comment:
It'd benefit the users by communicating this IMO. (The last two assume my other comment is accepted)
```suggestion

:::warning
Pivoting does some computations and overrides certain props based on those computations.
When pivot mode is active, these props will be ignored: `rows`, `columns`, `rowGroupingModel`, `aggregationModel`, `getAggregationPosition`, `columnVisibilityModel`, `columnGroupingModel`, `groupingColDef`, `headerFilters`, `disableRowGrouping`, and `disableAggregation`.
:::

```

Reply:
iterating on the copy above - I would avoid describing props as "user-provided" since they're more like "developer-provided" (as opposed to "end user-provided"), and we're generally speaking to "you the developer."

```suggestion

:::warning
Pivoting performs certain computations and uses them to override corresponding props.
When pivot mode is active, the following props are ignored: `rows`, `columns`, `rowGroupingModel`, `aggregationModel`, `getAggregationPosition`, `columnVisibilityModel`, `columnGroupingModel`, `groupingColDef`, `headerFilters`, `disableRowGrouping`, and `disableAggregation`.
:::

```